supabase/migrations/20240415132809_add_creation_timestamp_to_parcels.sql

```diff
@@ -0,0 +1,33 @@
+alter table "public"."parcels" add column "created_at" timestamp with time zone not null default (now() AT TIME ZONE 'utc'::text);
```

Comment:
does this default value result in the right time in terms of time zones? (time zones confuse me)

Reply:
niice